woocommerce/woocommerce-gutenberg-products-block
# Fix: only allow Mini Cart block in Widgets and Site Editor

Fixes #5576 

This PR prevents adding the Mini Cart block to posts, pages, or any custom post types by checking the `$pagenow`.




### Screenshots

![image](https://user-images.githubusercontent.com/5423135/151400049-14bd1ecf-e678-4f4d-8741-c40d0b51b178.png)



### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Edit a page
2. Try to add Mini Cart block, see no block found in the Inserter.
3. Go to Widgets (I used Storefront to test Widgets), see you can search and add Mini Cart to widget areas.
4. Using a block theme with Gutenberg enabled, go to the Site Editor, see you can search and add the Mini Cart block.
5. Using WP 5.9 with Gutenberg disabled, go to the Site Editor, see you can search and add the Mini Cart block.
### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
